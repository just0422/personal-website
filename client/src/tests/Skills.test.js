import React from 'react';
import {shallow} from 'enzyme';

import Skill from 'Resume/Skills';
import { skills } from './data';

describe('Skill tests', () => {
	it('should render correctly with no props', () => {
		const component = shallow(<Skill />);
		expect(component).toMatchSnapshot();
	});
	it('should render correctly with a skills prop', () => {
		const component = shallow(<Skill skills={skills} />);
		expect(component).toMatchSnapshot();
	});
});
