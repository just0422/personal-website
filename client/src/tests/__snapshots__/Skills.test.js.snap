// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skill tests should render correctly with a skills prop 1`] = `ShallowWrapper {}`;

exports[`Skill tests should render correctly with no props 1`] = `ShallowWrapper {}`;
